WLC/utils/azure.py

```diff
@@ -63,6 +63,25 @@ def save_image_to_azure(self, container, image):
 
         return True, hashed
 
+    def save_template_and_test(self, container, template_file, test_file):
+        self.create_container_not_exist(container)
+
+        hashed = hashlib.md5(template_file.tobytes()).hexdigest()
+
+        if self._block_blob_service.exists(container, hashed):
+            return '', 1
+
+        template_filename = '{}.py'.format(hashed)
+        test_filename = '{}.json'.format(hashed)
```

Comment:
The test file is just an array of test cases so json and not python. Please look at how the existing code which I wrote works.